src/properties.json

```diff
@@ -1,6 +1,6 @@
 {
     "matlabBatchUrl": "https://ssd.mathworks.com/supportfiles/ci/matlab-batch/v0/install.sh",
     "matlabDepsUrl": "https://ssd.mathworks.com/supportfiles/ci/matlab-deps/v0/install.sh",
-    "matlabLatestReleaseUrl": "https://ssd.mathworks.com/supportfiles/ci/matlab-release/v0/latest",
+    "matlabLatestReleaseUrl": "https://mw-ci-static-dev.s3.amazonaws.com/matlab-release-info/latest",
```

Comment:
Will change back before merging. 2 integ tests fail because of the current mpm issues with Windows & Mac.